third_party/amd/backend/compiler.py

```diff
@@ -197,7 +197,7 @@ def make_llir(src, metadata, options):
             paths = [path for (name, path) in options.extern_libs if amd.need_extern_lib(llvm_mod, name)]
             llvm.link_extern_libs(llvm_mod, paths)
 
-        llvm.optimize_module(llvm_mod, llvm.OPTIMIZE_O3)
+        llvm.optimize_module(llvm_mod, llvm.OPTIMIZE_O3, "amdgcn-amd-amdhsa")
```

Comment:
I think we can do `amd.TARGET_TRIPLE` here so we don't scatter the same string various places.